client/state/selectors/get-current-route-parameterized.js

```diff
@@ -0,0 +1,27 @@
+/** @format */
+
+/**
+ * Internal dependencies
+ */
+import getCurrentRoute from 'state/selectors/get-current-route';
+import { getSiteSlug } from 'state/sites/selectors';
+
+/**
+ * Returns the current route with site slug replaced by :site and site Id
+ * replaced by :siteid. Note that other parameters such as :domain are not currently
+ * supported.
+ *
+ * @param  {Object}  state  Global state tree
+ * @param  {Number}  siteId Site ID
+ * @return {?String}        path of site if known
```

Comment:
Maybe that's just me but I find this comment a bit misleading. What about using `current route with site parameters` or something along that line?